zipkin-server/README.md

```diff
@@ -271,6 +271,12 @@ prefixed system property:
 $ KAFKA_ZOOKEEPER=127.0.0.1:2181 java -Dzipkin.collector.kafka.overrides.auto.offset.reset=largest -jar zipkin.jar
 ```
 
+### Kafka 0.10+ Collector
```

Comment:
move this to a note under "This collector remains a Kafka 0.8.x consumer, while Zipkin systems update to 0.9+."? I think more people will notice it this way.